src/EditorFeatures/CSharpTest/Classification/SyntacticClassifierTests.cs

```diff
@@ -3605,5 +3605,48 @@ public void LoadDirective()
                 PPKeyword("load"),
                 String("\"file.csx\""));
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Classification)]
+        public void IncompleteAwaitInNonAsyncContext()
+        {
+            var code = @"
+void M()
+{
+    var x = await
+}";
+            TestInClass(code,
+                Keyword("void"),
+                Identifier("M"),
+                Punctuation.OpenParen,
+                Punctuation.CloseParen,
+                Punctuation.OpenCurly,
+                Keyword("var"),
+                Identifier("x"),
+                Operators.Equals,
+                Keyword("await"),
+                Punctuation.CloseCurly);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Classification)]
+        public void CompleteAwaitInNonAsyncContext()
+        {
+            var code = @"
+void M()
+{
+    var x = await;
```

Comment:
And calling methods:
```
void M(int x)
{
    M(await
}
```

What happens in these cases when brace completion has already put the `)` in?